rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
I believe this means your commonjs exports object will be frozen. You might want to use `default` or [prune via memory plugin](https://github.com/developit/preact-compat/blob/master/rollup.config.js#L27-L30).